src/cmd/package.go

```diff
@@ -143,6 +144,23 @@ func newPackageCreateCommand(v *viper.Viper) *cobra.Command {
 	return cmd
 }
 
+// Converts registry overrides to a structured type.
+// Input is of the following form:
+// []string{"docker.io/library=docker.example.com", "docker.io=docker.example.com"}
+func parseRegistryOverrides(overrides []string) ([]types.RegistryOverride, error) {
```

Comment:
I think there are a few validation gaps we can add some checking for in this function. Particularly empty strings that would evaluate `Source` or `Override` to be empty strings.